package.json

```diff
@@ -1,15 +1,23 @@
 {
   "name": "derby",
   "description": "MVC framework making it easy to write realtime, collaborative applications that run in both Node.js and browsers.",
-  "version": "3.0.0-beta.1",
+  "version": "3.0.0-beta.3",
   "homepage": "http://derbyjs.com/",
   "repository": {
     "type": "git",
     "url": "git://github.com/derbyjs/derby.git"
   },
   "main": "dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./parsing": "./dist/parsing/index.js",
+    "./templates": "./dist/templates/index.js",
+    "./App": "./dist/App.js",
+    "./lib/App": "./dist/App.js",
+    "./AppForServer": "./dist/AppForServer.js"
+  },
   "files": [
-    "dist/**/*.js"
+    "dist/"
```

Comment:
For the final publish, let's only publish the JS files for now, until we can get more testing on the newly built-in types.